tests/phpunit/tests/functions/TestConfigWpHome.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * Test the _config_wp_home function
+ *
+ * @group Functions
+ * @covers ::_config_wp_home
+ */
+class Test_for_config_wp_home extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 57179
+	 * @dataProvider _config_wp_home_dataset
+	 */
+	function test_config_wp_home( $url, $expected ) {
+		$this->assertSame( $expected, _config_wp_home( $url ) );
+	}
+
+	function _config_wp_home_dataset() {
+		return array(
+			'default'   => array(
+				'url'      => '/',
+				'expected' => '/',
+			),
+			'htpps'     => array(
```

Comment:
```suggestion
			'https'     => array(
```
I also suggest adding a `http` dataset too as the docblock for `_config_wp_home()` mentions local dev environments, which may use `http://` instead.